src/controllers/office.js

```diff
@@ -0,0 +1,55 @@
+/* eslint-disable max-len */
+import Office from '../models/office';
+import ValidateOffice from '../middlewares/helperoffice';
+
+const ControllerOffice = {
+  createOffice(req, res) {
+    // eslint-disable-next-line no-restricted-globals
+    if (isNaN(req.body.age)) {
```

Comment:
@vic3king what is the concept behind the age attribute for an office? Also pls follow the format specified for the response `id`, `type`, `name` (not officeName)
<img width="634" alt="screen shot 2019-01-28 at 6 51 05 pm" src="https://user-images.githubusercontent.com/26818808/51855495-b47b7100-232d-11e9-85a5-f70eccd89a7d.png">

Reply:
Ok upon futher review i can under the age concept. Nice addition. Pls take note of the response format specified

Reply:
To enable your code to be more readable you can use `ageLimit` instead of just `age` i believe that will be more approrpiate